app/helpers/admin_helper.rb

```diff
@@ -37,6 +37,10 @@ module AdminHelper
 
   PROFILE_ADMIN = { name: "config: profile setup", controller: "profile_fields" }.freeze
 
+  TECH_MENU_ITEMS = [
```

Comment:
This is a list of tech resources that we add. Eventually we may move things like the Sidekiq page to here as well.